include/pm/device_runtime.h

```diff
@@ -130,13 +136,33 @@ int pm_device_runtime_put_async(const struct device *dev);
  */
 bool pm_device_runtime_is_enabled(const struct device *dev);
 
+/**
+ * @brief Initialize a device state to #PM_DEVICE_STATE_SUSPENDED.
+ *
+ * By default device state is initialized to #PM_DEVICE_STATE_ACTIVE. In
+ * general, this makes sense because the device initialization function will
+ * resume and configure a device, leaving it operational. However, when device
+ * runtime PM is enabled, the device may not be resumed and the init function
+ * will just enable device runtime PM. If that is the case, this function can be
```

Comment:
This section is confuse, "init" is referencing the device initialization ? If so it should not enable device runtime there, only call this function to reflect the proper device state.